modules/platforms/cpp/ignite/client/detail/node_connection.h

```diff
@@ -115,6 +115,7 @@ class node_connection : public std::enable_shared_from_this<node_connection> {
             }
         }
 
+        m_logger->log_debug("Performing request: op=" + std::to_string(int(op)) + ", req_id=" + std::to_string(reqId));
```

Comment:
Does this affect performance even when debug logging is disabled? String concatenation is performed anyway, right?

Reply:
You are right. I'll fix it.